### PR TITLE
Channel notifications for Android Oreo or above

### PR DESCRIPTION
**Pull request by ckarmy**
Channel notifications implementation to receive notifications on android Oreo or above.  Since Google are going to stop accepting apps to the store which target at least Android Oreo. 

August 2018: New apps required to target API level 26 (Android 8.0) or higher.
November 2018: Updates to existing apps required to target API level 26 or higher.
2019 onwards: Each year the targetSdkVersion requirement will advance. Within one year following each Android dessert release, new apps and app updates will need to target the corresponding API level or higher.

https://github.com/fuse-open/fuselibs/issues/1187

Note that the `.unoproj` need to set the SDK at least 26 or update fuse to use sdk26 by default

> "SDK": {
>       "BuildToolsVersion": "26.0.1",
>       "CompileVersion": 26,
>       "MinVersion": 16,
>       "TargetVersion": 26
>     }

### CODE REVIEW

**Comment by jesusmartinoza** [comment]
Great @ckarmy! I'm agree with Luis to wait a little until we have the fuselibs patch.

File: src/Firebase.Notifications.Android/Android/PushNotificationReceiver.java [hunk 1]
```diff
@@ -253,9 +258,7 @@ void SpitOutNotification(RemoteMessage remoteMessage)
             n.defaults |= Notification.DEFAULT_SOUND;
         n.defaults |= Notification.DEFAULT_LIGHTS;
         n.defaults |= Notification.DEFAULT_VIBRATE;
-
         int id = PushNotificationReceiver.nextID();
-
         notificationManager.notify(id, n);
     }
-}
+}
```
Comment (jesusmartinoza) [inline src/Firebase.Notifications.Android/Android/PushNotificationReceiver.java]: Would be nice to remove this change and the one of line 238

File: src/Firebase.Notifications.Android/Android/PushNotificationReceiver.java [hunk 2]
```diff
@@ -230,7 +235,7 @@ void SpitOutNotification(RemoteMessage remoteMessage)
                     android.graphics.Bitmap bm = android.graphics.BitmapFactory.decodeStream(afs);
                     try
                     {
-                       afs.close();
+                        afs.close();
```
Comment (jesusmartinoza) [inline src/Firebase.Notifications.Android/Android/PushNotificationReceiver.java]: And this one :)

**Comment by LuisRodriguezLD** [comment]
closing as #87 fixes the issue

**Pull request closed without merging**